ViSearch.user.js

```diff
@@ -159,9 +168,10 @@ s.type = "text/css";
 s.innerHTML = styleSheet;
 (document.head || document.documentElement).appendChild(s);
 
-window.addEventListener('load', function() {
+// window.addEventListener('load', function() {   //是否可以listen别的
+    window.addEventListener('scroll', function() {  //改成滚动的时候就触发？或者滚动多次
```

Comment:
Why use scroll instead of load here?

Reply:
Testing auto Next page to refresh D3 data